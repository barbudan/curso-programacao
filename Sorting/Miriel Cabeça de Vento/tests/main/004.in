13
cxyz blablablab blebleble
abc defgabc defgabc defg
dddoopz kaos owiw mam
abc dejfabc dejfabc dejf
dddoopa kaos owiz mam
bcd efzbcd efzbcd efzbcd efz
bcd efbcd efbcd efbcd ef
dddoopz kaoz owiw mam
cxyz blablabla blebleble
bcd febcd febcd febcd fe
cxzy oyoyoy eiei kkk
dddoopa kaos owiw mam
cxzy oyoyoy eiei kkz