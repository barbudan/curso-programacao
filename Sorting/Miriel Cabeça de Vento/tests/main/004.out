abc defgabc defgabc defg
abc dejfabc dejfabc dejf
bcd efbcd efbcd efbcd ef
bcd efzbcd efzbcd efzbcd efz
bcd febcd febcd febcd fe
cxyz blablabla blebleble
cxyz blablablab blebleble
cxzy oyoyoy eiei kkk
cxzy oyoyoy eiei kkz
dddoopa kaos owiw mam
dddoopa kaos owiz mam
dddoopz kaos owiw mam
dddoopz kaoz owiw mam
